ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.ts

```diff
@@ -50,8 +50,11 @@ export class StringSequencePartComponent implements OnInit {
       this.partValidation?.maxLength ?? Number.MAX_SAFE_INTEGER,
       this.partValidation?.minLength ?? 1,
     );
+    this.setDefaultValue();
+  }
 
-    if (!this.value) this.modelChanged(this.partValidationSafe.isRequired ? [''] : []);
+  ngOnChanges(changes: SimpleChanges): void {
```

Comment:
Why is this needed now?

Reply:
Because child components in job.component.html in ngSwitch are not destroyed when you switch between hyperdrive and hyperdrive form config.